### ISSUE TITLE
missed dependency for YAML

### ISSUE THREAD

**Issue by KES777** [frioux/DBIx-Class-DeploymentHandler#71]
on clean system I get next error when run `prepare` command:

     Can't load producer 'SQL::Translator::Producer::YAML' : Error loading SQL::Translator::Producer::YAML as SQL::Translator::Producer::YAML : Can't locate YAML.pm in @INC (you may need to install the YAML module)

My `cpanfile` is:

```
requires 'Carton';
requires 'Mojolicious';
requires 'Mojolicious::Plugin::PODRenderer';
requires 'DBD::Pg';                   # by Database
requires 'DBIx::Class', '==0.082841';
requires 'DBIx::Class::Migration';
```

https://github.com/jjn1056/DBIx-Class-Migration/issues/123#event-2389434865

**Comment by mohawk2** [comment]
`YAML` needs to be a dep for this distro.

**Comment by mmcclimon** [comment]
Huh! I'm a bit surprised we never noticed (and that it's only an  optional dependency for SQL::Translator.) I'll get a new release out soon with proper deps.

**Comment by mmcclimon** [comment]
Thanks for the report; I just added 296dff46 and released as v0.002232.

**Comment by KES777** [comment]
>Huh! I'm a bit surprised we never noticed (and that it's only an optional dependency for SQL::Translator.) I'll get a new release out soon with proper deps.

:sunglasses:
